packages/editor/src/dataviews/types.ts

```diff
@@ -1,5 +1,5 @@
 type PostStatus =
-	| 'published'
```

Comment:
I just realized we had this in the editor that we also have in the `wordpress/fields` package ([see](https://github.com/WordPress/gutenberg/blob/6d36c1b8ee1c3c68601a61959bc1c2602ee107d1/packages/fields/src/types.ts#L1)). It sounds to me the fields package should be the central package for that kind of information about fields and posts. Not for this PR, just sharing a potential follow-up to clean things up later.